editor/src/components/editor/actions/actions.tsx

```diff
@@ -6319,6 +6322,7 @@ export async function load(
   // this action is now async!
   const migratedModel = applyMigrations(model)
   const npmDependencies = dependenciesWithEditorRequirements(migratedModel.projectContents)
+  notifyOperationStarted(dispatch, { type: 'refreshDependencies' })
```

Comment:
in actions.tsx, please mark all side effects with a comment to make future debugging easier – by default we assume the update functions are pure

Reply:
thanks! marked.